src/app/App.tsx

```diff
@@ -11,6 +15,9 @@ import { AppState } from "./AppState";
 import Settings from "./components/Settings";
 import Wallet from "./components/Wallet";
 
+const remote = window.require("electron").remote;
+let fullScreen = false;
```

Comment:
the app stores you last screen size, initializing this to false leads to incorrect behaviors

Reply:
Fixed!. Checking actual window size to initialize variable